configs.js

```diff
@@ -222,6 +222,23 @@ function loadStandaloneConfig() {
 
         metrics = conf.metrics;
 
+        if (_.isArray(configs.ignore)) {
+            let filtered = metrics;
+
+            configs.ignore.forEach((metric) => {
+                // eslint-disable-next-line array-callback-return
+                filtered = filtered.filter(function (Metric) {
```

Comment:
Why don't you use an arrow function here?

Reply:
done